### ISSUE TITLE
Remapped keys on Debian not recognised correctly

### ISSUE THREAD

**Issue by ani-rbnm** [microsoft/vscode#227133]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 1.92.2
- OS Version: Debian 12

Steps to Reproduce:

1. A bunch of modifier keycodes on the keyboard are mapped to alternative ones at the OS level (Xmodmap). One example is the "alt" key mapped to "ctrl".
2. keyboard.dispatch: "keyCode" set in settings.json. 
3. Testing keymappings with "Developer: Toggle Keyboard Shortcuts Troubleshooting", we see that "alt" correctly maps to "ctrl" when used by itself.
4. As soon as it is pressed with another key, it resets back to the original value.
In other words,
press "alt" -> outputs "ctrl" correctly
press "alt + a" -> prints "alt + a"

```[tasklist]
### Tasks
```

**Comment by ani-rbnm** [comment]
screengrab of the issue replicated on debian 12

https://github.com/user-attachments/assets/d2bab190-79d4-4201-9178-e1a6447d8f02
